components/Comment/CommentAnonymously.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from "react";
+import Switch from "@mui/material/Switch";
+import { css, StyleSheet } from "aphrodite";
+
+type Props = {
+  id: string;
+  anonymous: boolean;
```

Comment:
just a nit code practice stuff that we try to start flags with verbs
```suggestion
  isAnonymous: boolean;
```